### DISCUSSION TITLE
New translation in Italian

### DISCUSSION THREAD

**Discussion by adrianodaddiego** [sircharlo/meeting-media-manager#3527] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak: Italiano/Italian} - I/it

**Comment by mtdvlpr** [comment]
Thank you for your willing spirit! Please [sign in to Crowdin](https://crowdin.com/project/meeting-media-manager) using your GitHub account and request to join our translation project.

**Comment by mtdvlpr** [comment]
Hi @adrianodaddiego, are you [nicerose78](https://crowdin.com/profile/nicerose78)?

**Comment by adrianodaddiego** [comment]
Hi, no i'm adrianodaddiego

Il lun 25 nov 2024, 15:30 Manoah Tervoort ***@***.***> ha
scritto:

**Reply by mtdvlpr** [reply]
Were you able to create a Crowdin account and request access?

**Comment by adrianodaddiego** [comment]
Do it! Thank you

Il giorno lun 25 nov 2024 alle ore 17:22 Manoah Tervoort <
***@***.***> ha scritto:

**Reply by mtdvlpr** [reply]
I've accepted your request! Thank you for your help. If you have questions, don't hesitate to ask.

**Comment by mtdvlpr** [comment]
Hi @adrianodaddiego, thank you for your hard work! I have good news. Italian will be included in the next release that will be released this week 😃

**Comment by adrianodaddiego** [comment]
Thank you! I will refine the translation while using.

Adriano

Il giorno lun 2 dic 2024 alle ore 15:35 Manoah Tervoort <
***@***.***> ha scritto:
